UnitySDK/Assets/ML-Agents/Scripts/Batcher.cs

```diff
@@ -88,18 +87,9 @@ public Batcher(ICommunicator communicator)
             }
             catch
             {
-                var exceptionMessage = "The Communicator was unable to connect. Please make sure the External " +
-                                       "process is ready to accept communication with Unity.";
-
-                // Check for common error condition and add details to the exception message.
-                var httpProxy = Environment.GetEnvironmentVariable("HTTP_PROXY");
-                var httpsProxy = Environment.GetEnvironmentVariable("HTTPS_PROXY");
-                if (httpProxy != null || httpsProxy != null)
-                {
-                    exceptionMessage += " Try removing HTTP_PROXY and HTTPS_PROXY from the" +
-                                        "environment variables and try again.";
-                }
-                throw new UnityAgentsException(exceptionMessage);
+                throw new UnityAgentsException(
+                    "The Communicator was unable to connect. Please make sure the External " +
+                    "process is ready to accept communication with Unity.");
```

Comment:
Why are we changing this?

Reply:
not sure. was this something you changed during your refactor @surfnerd ?

Reply:
I think @chriselion added this to have a more helpful error message.

Reply:
Yeah, looks like it undoes the change.